_layouts/empty.html

```diff
@@ -0,0 +1 @@
+{{ content }}
```

Comment:
It's possible to not specify a layout, but I was having trouble with getting the different rules to work out - you have the default as recipes. I think we could change that to default for posts, but it wasn't playing nice.